doc/Overview/Reactive/How-tos/SimpleFeed.md

```diff
@@ -1,30 +1,31 @@
 ---
-uid: Overview.Reactive.HowTos.SimpleFeed
+uid: Overview.Reactive.SimpleFeed
 ---
 
 # How to create a simple feed
 
-In this tutorial you will learn how to create an MVUX project and use a combination of a feed (`IFeed<T>`) and the `FeedView` control to load and display data coming from a service.
+In this tutorial you will learn how to create an MVUX project and use a combination of a feed (`IFeed<T>`)
+and the `FeedView` control to asynchronously load and display data coming from a service.
```

Comment:
I get that md supports random line breaks but I don't like this and would prefer to just use an editor that supports word wrapping!

Reply:
The reason I like wrapping MDs in the file, is for when it's viewed on GH it's irritating.
I can fix them all if you want me to. I'll def avoid from now on.